conda_smithy/lint_recipe.py

```diff
@@ -256,6 +257,14 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
                     lints.append("`noarch: python` packages should use pip. "
                                  "See https://conda-forge.org/docs/meta.html#use-pip")
 
+    # 19: check version
+    if package_section.get('version') is not None:
```

Comment:
Is there ever a situation where we don't have a version in the `package_section`?

Reply:
There can be a situation where it is not in a bad recipe.